src/resources/elements/primeDesignSystem/pbutton/pbutton.scss

```diff
@@ -3,7 +3,8 @@
 @mixin standardBackground {
   border: none;
   padding: 14px 28px;
-  background: $Gradient01;
+  background-image: $Gradient08;
```

Comment:
Can we make the animation be an optional behavior?

First, are you sure that all three button types are supposed to support the animation?  My bet would be only Primary.  You have only shown the Primary one in the screen shot in this PR.

In any case, I would like to be able to display any and all button types without an animation, simply by changing a property on the button, like a boolean `animate-bk`.

Reply:
Quick question- by animation, do you refer to the behavior of the button or the transition effect?
i.E. - 
For the primary, the behavior is changing the gradient bg. 
For the secondary and tertiary it changes the bg color.
on click all translate 1px up.

Should these behaviors be disabled by the new property or only the smooth transition animation?
I've added a property `do-not-animate`, which currently disable all user interactions.

The primary hover animation introduced here is requested in the design system Figma:
![image](https://user-images.githubusercontent.com/2517870/147849709-e18dbf88-4791-415b-97d6-1d659902cfff.png)

That being said- The `spinning loading icon` behavior is still absent.